components/eam/src/physics/cam/zm_conv.F90

```diff
@@ -100,7 +127,16 @@ module zm_conv
    
    integer,protected ::  limcnv   ! top interface level limit for convection
 
-   real(r8) :: tp_fac = unset_r8  ! PMA tunes tpert 
+   real(r8) :: tp_fac = unset_r8  ! PMA tunes tpert
```

Comment:
what does "PMA" stand for? Are these someone's initials? if so we should remove it.

Reply:
"PMA" stands for "Po-Lun Ma", which is now removed in the comment.